llvm/utils/TableGen/Common/GlobalISel/GlobalISelMatchTable.cpp

```diff
@@ -1593,7 +1594,7 @@ void MemoryAlignmentPredicateMatcher::emitPredicateOpcodes(
   Table << MatchTable::Opcode("GIM_CheckMemoryAlignment")
         << MatchTable::Comment("MI") << MatchTable::ULEB128Value(InsnVarID)
         << MatchTable::Comment("MMO") << MatchTable::ULEB128Value(MMOIdx)
-        << MatchTable::Comment("MinAlign") << MatchTable::ULEB128Value(MinAlign)
+        << MatchTable::Comment("MinAlign") << MatchTable::IntValue(1, MinAlign)
```

Comment:
Maybe assert `MinAlign < 256` somewhere? This is not log alignment, so it would break if some vector load instruction actually required 256-byte or greater alignment.